src/Robo/Plugin/Commands/EnvSwitchCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Fire\Robo\Plugin\Commands;
+
+use Fire\Robo\Plugin\Commands\FireCommandBase;
+use Robo\Symfony\ConsoleIO;
+use Robo\Robo;
+
+/**
+ * Provides a command to Switch between the local_environment.
+ */
+class EnvSwitchCommand extends FireCommandBase {
+
+  /**
+   * Add a new command.
+   *
+   * Usage Example: fire switch
+   *
+   * @command env:switch
+   * @aliases env-switch, switch, sw
+   * @usage fire switch
+   */
+  public function switch(ConsoleIO $io) {
+    $tasks = $this->collectionBuilder($io);
+    $env = Robo::config()->get('local_environment');
+    $new_env = ($env === 'lando') ? 'ddev' : 'lando';
+    $filename = 'fire.local.yml';
```

Comment:
please use the full path to the file, if you do it like this and you run the command from other folder that is not the root project, you will have a error, the FireCommandBase already have a function that returns to you the projects root folder path

Reply:
Hi @rigoucr done!